lib/index.js

```diff
@@ -163,7 +155,7 @@ class Linter {
 
     for (let rule of rules) {
       try {
-        traverse(templateAST, buildPlugin(rule));
+        transform(templateAST, () => rule.getVisitor());
```

Comment:
Ya, wasn’t sure if you’d still want / need `buildPlugin` (now that `rule.templateEnvironmentData` isn’t used). Glad to see this simplified. :)